packages/edit-site/src/components/sidebar/global-styles-v2/components/screen-header.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	Button,
+	Heading,
+	HStack,
+	Icon,
+	NavigatorLink,
+	Spacer,
+	Text,
+	View,
+	VStack,
+} from '@wordpress/components';
+import { chevronLeft } from '@wordpress/icons';
```

Comment:
We're using `@wordpress/icons` instead of `react-icons` for any icons in this prototype.